### ISSUE TITLE
Map tilt after switching to offline map possible even when disabled in settings

### ISSUE THREAD

**Issue by ThomGr** [cachly/issues-tracker#5082]
**Describe the bug**
When switching to offline maps, map tilt by two finger swipe up is possible even when permanently disabled in settings. After switching to another screen and coming back tilting is correctly blocked. Does not happen everyone, though...

**To Reproduce**
I see this quite often, but cannot clearly reproduce

**Expected behavior**
Tilt should never be possible when disabled in settings 

**Screenshots**

**Cachly version**
 - Version: [e.g. 8.0.6 (1)]

**iOS device (please complete the following information):**
 - Device: [e.g. iPhone 14]
 - OS: [e.g. iOS 13.1]

**Additional context**

**Comment by cachly** [comment]
Which specific map screen do you see this happening? Do you have a specific interface open such as Lists > Offline List, Cache > Navigate to cache?

**Comment by ThomGr** [comment]
This was in navigating to a cache, offline maps (release, not test of new maps), possibly offline list but may have been live. I just wonder if iOS reports location to Cachly sticky to roads which only makes sense for car navigation. Am 03.11.2024 um 20:34 schrieb Nicholas Hubbard ***@***.***>:﻿
Which specific map screen do you see this happening? Do you have a specific interface open such as Lists > Offline List, Cache > Navigate to cache?

**Comment by cachly** [comment]
This was specifically about the map tilt bug. I cannot replicate.

**Comment by ThomGr** [comment]
Yeah, close it, will open again when I can replicateAm 03.11.2024 um 21:25 schrieb Nicholas Hubbard ***@***.***>:﻿
This was specifically about the map tilt bug. I cannot replicate.
